frontend/src/components/form/index.tsx

```diff
@@ -29,19 +33,88 @@ Form.Title = function FormTitle({ children, ...restProps }: any) {
   return <Title {...restProps}>{children}</Title>
 }
 
-Form.Input = function FormInput({ setInput, ...restProps }: any) {
+Form.Input = function FormInput({ handleInputChange, ...restProps }: any) {
   return (
     <Input
       marginY={[1, 2]}
       padding={[1, 2]}
       {...restProps}
-      onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-        setInput(event.target.value)
-      }
+      onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+        console.log(event.target.value)
+        handleInputChange(event)
+      }}
     />
   )
 }
 
+export const InputField = ({
```

Comment:
These components should be in separate files. They could still be imported from this file and exported through "Form":
`Form.Input = InputField`

Reply:
Pull "register" function (functionality) outside of this component:
`<InputField {...register(name, validation)} />`

Reply:
Ne znam kako bi se ovo odradilo, pokusala sam da ga izbacim i onda mi je register bacao neke greske pa ga nisam dirala